### ISSUE TITLE
feat:(forge) Size of contracts via 'forge size' or 'forge build --size'

### ISSUE THREAD

**Issue by pbshgthm** [foundry-rs/foundry#588]
It'll be useful to have a command that prints out the contact sizes in kb. This would help optimise contract size and stay below the 24kb limit, especially while importing multiple libraries. The command could look something like this

```
$ forge size

SampleContract       :      23kb
AnotherContract      :       2kb
NewContract          :      16kb
```

Alternatively, this feature can also be part of forge build with a tag, something like

```
$ forge build --size

compiling...
success.

SampleContract       :      23kb
AnotherContract      :       2kb
NewContract          :      16kb
```

If you think this could be a useful addition to forge, I can take this up.

**Comment by odyslam** [comment]
Absolutely, I created a script for dapptools-template and I remember it was widely accepted as a good idea. 

On top of that, we could use a gas price API to show the approximate price of deploying the contract at current gas market conditions.

Here is the bash script that did all of that: https://github.com/gakonst/dapptools-template/blob/master/scripts/common.sh

**Comment by pbshgthm** [comment]
Awesome, the gas price would be an interesting feature too. I'll look into it.

**Comment by onbjerg** [comment]
Was this addressed in #682?

**Comment by onbjerg** [comment]
Closing this because of #682
